### PR TITLE
[WIP] Add Azure DevOps-as-CI Documentation

### PR DESCRIPTION
**Pull request by echesakovMSFT**

### CODE REVIEW

**Comment by echesakovMSFT** [comment]
cc @dotnet/jit-contrib

File: Documentation/azure-devops-as-ci/azure-devops-as-ci.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+since they are using limited hardware.
+
+The results for those jobs can be found in the new CI system. 
+Other Jenkins jobs are running as usual and we will continue monitoring them to validate consistency between Jenkins and AzDev test jobs.
+
+# How to find the test results in Azure DevOps?
+
+If you click on coreclr-ci Details link 
+![coreclr-ci-badge](coreclr-ci-badge.png)
```
Comment (jkotas) [inline Documentation/azure-devops-as-ci/azure-devops-as-ci.md]: Do we really need MB worth of pictures in the repo - these pictures will be obsolete in a few weeks or months as the UI develops.

Comment (echesakovMSFT) [reply Documentation/azure-devops-as-ci/azure-devops-as-ci.md]: I agree we don't need all those pictures and the only reason why I opened this is to have a way for contributor to find their test result in the new CI system while we are testing and adjusting it.

**Pull request closed without merging**